organising/organising-a-meeting.md

```diff
@@ -0,0 +1,164 @@
+
+## Schedule the meeting
+
+This means making sure we know when the meeting is.
+
+Our standard schedule is the 2nd Monday of the month, but sometimes we need to move it around, if it clashes with a bank holiday, or some other big event.  If we do need to move it shifting to the next or previous Monday is the preferred option, but work out what is most convenient with the rest of the organising team.
+
+## Confirm the speakers
+
+Once we know the date, we can start confirming speakers.  We have a [spreadsheet of volunteers][schedule-preadsheet] where we maintain a list of current volunteers, with notes and pencilled in dates.  There are links into [our shared inbox software][shared-inbox] to the conversations with each volunteer so you can chase up the speakers.
+
+### What we need from them
+
+To confirm the speakers we need:
+
+* __Their name__ - how we should refer to them on the site
+* __Their talk title__ - what their talk will be called
+* __A short description of the talk__ - a sentence of two to entice people to see the talk.  Let them know we use markdown so they can format it and sprinkle in relevant links.
+* __A preferred timeslot__ - From 10, 25, or 40 minutes, refer to [our notes on timing](/#timing) from the main readme.  This helps us budget the agenda.
+* __A preferred url for linking to__ - in the write up we'll link the speakers name to this so folk can find out more, usually it's twitter, github, a personal blog, or linkedin.
+
+### A full agenda
+
+We run the meetings from 6:30pm to 8:00pm.  With 10 minutes or so admin at the start, that means we have about 1 hour and 20 minutes (80 minutes) to fill.  Combinations that work well:
+
+* one 10 minute talk, one 25 minute talk, one 40 minute talk (~75 minutes)
+* three 25 minute talks (~75 minutes)
+* two 25 minute slots, two 10 minute slots (~70 minutes)
+
+Combinations that work, but are painful:
+
+* two 40 minute talks (~80 minutes) - 40 minute talks can drag on a bit, we try to avoid this, but sometimes this is what we
+
+Special combinations:
+
+* eight 10 minute talks (~80 minutes) - usually this is what we do for our February lightning talks only event.  It's a fun evening, but tough to organise so we don't do it that often.  If you have a glut of speakers though, by all means!
+* one 80 minute session - for a practical event, or panel discussion, or pub quiz, or something.  We wouldn't want a single speaker to talk for 80 minutes, but if someone has an idea for a interactive session that would take up the whole time then why not try it!
+
+But!  Any combination that fills roughly 80 minutes will do.  It's also fine to not fill 80 minutes.  Getting one speaker in just so we can make an LRUG happen is good.
```

Comment:
I think we should move this line up nearer the top -- it's very rare at the moment that we "fill" the time, and I wouldn't want anyone new feeling like they should expect to. The "combinations that work well" a few paragraphs up are more the exception than the rule (but still useful to include).